KubeArmor/config/config.go

```diff
@@ -103,6 +104,7 @@ const (
 	ConfigAlertThrottling                string = "alertThrottling"
 	ConfigMaxAlertPerSec                 string = "maxAlertPerSec"
 	ConfigThrottleSec                    string = "throttleSec"
+	ConfigControllerPresent              string = "controllerPresent"
```

Comment:
NIT: `controllerPresent` doesn't sound very intuitive. How about something like `annotateResources` which sounds more specific?
If users want to skip annotation in KA they say `annotateResources=false`